tests/cephfs/cephfs_utilsV1.py

```diff
@@ -69,7 +70,20 @@ def __init__(self, ceph_cluster, **kwargs):
         self.osds = ceph_cluster.get_ceph_objects("osd")
         self.mdss = ceph_cluster.get_ceph_objects("mds")
         self.clients = ceph_cluster.get_ceph_objects("client")
+        self.installer = ceph_cluster.get_nodes(role="installer")[0]
         self.test_data = kwargs.get("test_data")
+        enable_log = (
+            FsUtils.get_custom_config_value(self.test_data, "enable_log")
+            if self.test_data
+            else False
+        )
+        if enable_log:
+            # option 1
+            cephadm = CephAdm(self.installer)
+            cephadm.ceph.config.set(key="log_to_file", value="true", daemon="global")
```

Comment:
Hi @AmarnatReddy as discussed please check if we can replace this line in existing enable_logs module in same testlib, in order to maintain one module to invoke for enable/disable debug logs, suggesting to use the existing one because, it can be used to enable any daemon debug logs plus configure for debug levels too,

sample:
-
    test:
      abort-on-fail: false
      desc: "Enable ceph debug logs"
      module: cephfs_logs_util.py
      name: cephfs-enable-logs
      config:
       ENABLE_LOGS : 1
       daemon_list : ['mds','client']
       daemon_dbg_level : {'mds':20,'client':20}

-
    test:
      abort-on-fail: false
      desc: "Disable ceph debug logs"
      module: cephfs_logs_util.py
      name: cephfs-disable-logs
      config:
       DISABLE_LOGS : 1
       daemon_list : ['mds','client']

Reply:
Hi Suma, 
We plan to implement this at the custom-config level, where users will need to pass these values as part of the run.py command.

If we want to implement it at the test case level, more changes would be required in utilsV1, as we’re not currently passing test case config information to the utilsV1 file. This may need to be handled directly within the test case itself.